loc/en.js

```diff
@@ -382,6 +382,7 @@ module.exports = {
       address: 'address',
       address_field_is_not_valid: 'Address field is not valid',
       amount_field_is_not_valid: 'Amount field is not valid',
+      amount_is_too_high: "There is a maximum of 21,000,000 BTCV that will ever exist, so don't try to send more!",
```

Comment:
would be nice if 21,000,000 would be as the variable

Reply:
done